src/main/cli/process.ts

```diff
@@ -140,5 +140,19 @@ export function cli(mainFct: () => void) {
             },
         )
         .help()
-        .parse((_PACKAGING === "0") ? process.argv.slice(2) : process.argv.slice(1));
+        .parse(cliParse());
+}
+
+function cliParse() {
+    process.argv.shift();
+
+    let findIdx = 0;
+    while (process.argv[findIdx++].startsWith("--")) {
+        process.argv.shift();
```

Comment:
I do not think this code achieves the desired result. Test case:
```
node -e "const arr = ['--first', '--second', 'param1', 'param2', '--third', '--fourth']; let findIdx = 0; while (arr[findIdx++].startsWith('--')) { arr.shift(); } console.log(JSON.stringify(arr));"
```
=>
```
["--second","param1","param2","--third","--fourth"]
```

To make things simple, I think that the Electron/Chromium-injected `argv` parameters (which can occur anywhere in the CLI sequence of parameters, not just at the start) should be preserved, and we should just ignore them in the CLI processing (unrecognised by the Yargs schema => ignore).

Reply:
I see, But how to do that ? 

yargs parse : --help and --version
Should we filter only help and version option and ignore all '--' option ?

Reply:
If I am not mistaken, Yargs is just a convenient wrapper around 'argv' which effectively allows us to define (in a declarative manner) a whitelist of parameters, as well as a catch-all fallback procedure for unrecognised params (in our case, we filter-in EPUB paths). So I think we should be able to filter-out any non-recognized params (i.e. outside of the declared whitelist) in Yargs' fallback / catch-all callback?

Reply:
I just saw your message, I propose a new fix
What do you think ?

Reply:
I tested my code on linux